tests/testthat/test-hipercow.R

```diff
@@ -1,3 +1,5 @@
 test_that("hipercow", {
+  mock_play <- mockery::mock(FALSE)
+  mockery::stub(hipercow, "hipercow_speak", mock_play)
   expect_output(hipercow(), "H.+I.+P.+E.+R.+C.+O.+W")
```

Comment:
Do you want to check that play was called here?

Why are you adding a `FALSE` return value to your mock function? I think you simply want `mock_play <- mockery::mock()` here and in the previous

Reply:
Oops - copy/paste error on the FALSE. Not very much logic around the plays, but I've added expected call counts.